src/cpu_info.rs

```diff
@@ -72,10 +72,7 @@ pub fn get_cpu_info() -> (CpuManufacturer, InstructionSet) {
         }
 
         // This ensures we always use hardware intrinsics and we do not use software emulation
-        #[cfg(all(
-            any(target_arch = "x86", target_arch = "x86_64"),
-            target_feature = "sse2"
-        ))]
```

Comment:
Since we are not passing anymore the sse2 and avx2 features to the compiler, if we keep target_feature="sse2" the following code would not be compiled. Also, if we always compile that code there should be no problem since it is using the ```cpuid``` instruction which is available on x86 and x86_64 architecture.
As a further improvement, the feature detection provided by cpu_info.rs could be replaced by the one in the stdd:arch crate (https://doc.rust-lang.org/1.29.1/std/arch/index.html)
I agree that the ```all()``` can be removed